cmake/modules/3.16/FindPostgreSQL.cmake

```diff
@@ -10,6 +10,8 @@ Find the PostgreSQL installation.
 IMPORTED Targets
 ^^^^^^^^^^^^^^^^
 
+.. versionadded:: 3.14
```

Comment:
Ah good to see that the newest upstream version of FindPostgreSQL.cmake handles this cleanly, but looking at the history, the addition of the server component dates to commit https://github.com/Kitware/CMake/commit/8b066f1a65466ae2dc542b097f877795f386f942 which went to 3.20.  So this file should be move to a 3.20 subdirectory with the appropriate changes to deal with that version.

Reply:
Good catch. I will deal with this tonight.